### PR TITLE
PLENTY-220_fix_shopbuilder_filter_button

### PR DESCRIPTION
**Pull request by TheKeymaster**

### CODE REVIEW

File: resources/scss/shop-builder.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// This file overwrites standard plenty shop builder styles. All styles found in this file only
+// have an effect when using the shop builder. The styles are always loaded, even if the shop builder is not used.
```
Comment (dblaichinger) [inline resources/scss/shop-builder.scss]: > All styles found in this file only have an effect when using the shop builder. The styles are always loaded, even if the shop builder is not used.

That is a bit misleading, if the styles are always loaded even without shop builder, how can those style only have effect on the shop builder? Are those classes only available when using shop builder?

Comment (TheKeymaster) [reply resources/scss/shop-builder.scss]: > Are those classes only available when using shop builder?

Congratulations, you are a millionaire! Yes those classes are only available when using Shop Builder and I want to keep it that way for future issues. Everything that is only-shopbuilder should stay in here and should only have an effect when using only-shopbuilder.

Comment (dblaichinger) [reply resources/scss/shop-builder.scss]: Please update your comment to make it more clear, when you stumble across this file the first time its confusing.

Comment (TheKeymaster) [reply resources/scss/shop-builder.scss]: I hope it's more clear now.

**Pull request merged**